README.md

```diff
@@ -132,10 +132,25 @@ If you wish, you can manually delegate exceptions, e.g. `try { ... } catch(e) {
 
 ## Setup for ReactJS
 
-Follow the general instructions denoted in _Setup for JavaScript_ to load and initialize the library.  
+Follow the general instructions denoted in _Setup for JavaScript_ to load and initialize the library.
 
 There is nothing specific that needs to be done with React, other than making sure to initialize the library in your root entry point(typically `index.js`).
 
+## Configuring without an API key
+
+If you are in a situation where an API key is not an option, you can configure the endpoint that errors are sent to with the following:
+
+```javascript
+const errorHandler = new StackdriverErrorReporter();
+errorHandler.start({
+  targetUrl: '<my-custom-url>',
+  service: '<my-service>',              // (optional)
+  version: '<my-service-version>'       // (optional)
+});
+```
+
```

Comment:
Please detail more what's expected to be put in `targetUrl`: is it compatible with relative URLs or should I provide an absolute one?